Configure

```diff
@@ -650,7 +669,7 @@ while (@argvcopy)
                         $disabled{$1} = "option";
                         }
 		# No longer an automatic choice
-		$auto_threads = 0 if ($1 eq "threads");
+		$auto{$1} = 0;
```

Comment:
I don't quite follow this, $1 can be anything here... Is it intentional that you set say $auto{zlib}?

Reply:
Yes.  For the use now, `$auto{zlib} = 0` says that the zlib option has been configured manually, and well, I don't see the harm.

Reply:
Trouble with this of course is that it's challenging to tell cases like zlib apart. I mean normally you'd just `if (!$auto{zlib})`, but as it stands now you'd have to `if (defined($auto{zlib}) && !$auto{zlib})`. With this in mind I'd say that it would be appropriate to have some condition, e.g. `$auto{$1} = 0 if (defined($auto{$1}))`. This way at least it would be less confusing as it would indicate that %auto can have only specific keys.

Reply:
Yeah, ok, you're right, quite unecessary to cause confusion.  Fixing now.